docs/proposals/fluentd-secure-forward.md

```diff
@@ -0,0 +1,35 @@
+# Allow Fluentd to secure forward to other Fluentd/Logstash (over ES-Copy)
+
+## Why?
+
+* Currently we allow forwarding just to another unsecure Elasticsearch
+* No easy way to ship off Fluentd collected logs for OSE to other log aggregators
+* More simple to support single plugin to do this than adding a number of Fluentd output plugins
+
+## How?
+
+Ship Fluentd image with (https://github.com/tagomoris/fluent-plugin-secure-forward)[fluent-plugin-secure-forward]
+and provide a means to configure it with configmaps/environment variables.
+
+## Proposal
+
+In lieu of using ES copy, which allows us to just forward a copy of our logs to
+another, unsecured, Elasticsearch cluster, we will provide a way for customers
+to configure and use the `Secure Forward` Fluentd plugin to enable them to integrate
+logs from OSE into their current log aggregation solution.
+
+# Limitations
+
+The customer would need to have something that can talk with the Fluentd plugin.
+
+* For Fluentd that means using the Secure Forward input plugin.
+* For Logstash, it looks like it has a Fluent codec [1] that may be able to be
+used in conjunction with secure TCP configurations [2].
```

Comment:
It looks like Fluentd has two splunk output plugins [1] [2]  I'd imagine a customer would set up a Fluentd instance that connected to their Splunk instance and would set up the OSE Fluentd to forward to that Fluentd instance.

[1] https://github.com/parolkar/fluent-plugin-splunk
[2] https://github.com/gtrevg/fluent-plugin-splunk-ex